lti_consumer/static/js/xblock_lti_consumer.js

```diff
@@ -149,14 +149,29 @@ function LtiConsumerXBlock(runtime, element) {
         var $element = $(element);
         var $ltiContainer = $element.find('.lti-consumer-container');
         var askToSendUsername = $ltiContainer.data('ask-to-send-username') == 'True';
+        var askToSendFullName = $ltiContainer.data('ask-to-send-full-name') == 'True';
         var askToSendEmail = $ltiContainer.data('ask-to-send-email') == 'True';
         var ltiVersion = $ltiContainer.data('lti-version');
 
         function renderPIIConsentPromptIfRequired(onSuccess, showCancelButton=true) {
-            if (askToSendUsername && askToSendEmail) {
+            if (askToSendUsername && askToSendFullName && askToSendEmail) {
```

Comment:
I can't dynamically construct this string because it makes translating the strings more difficult, so a series of if/elses is best.

Reply:
Ah that makes sense. Thanks for clarifying